Src/ILGPU/ILGPU.csproj

```diff
@@ -50,54 +51,54 @@
             <DependentUpon>AtomicFunctions.tt</DependentUpon>
         </None>
         <None Include="Backends\PTX\PTXIntrinsics.Generated.cs">
-          <DesignTime>True</DesignTime>
-          <AutoGen>True</AutoGen>
-          <DependentUpon>PTXIntrinsics.Generated.tt</DependentUpon>
+            <DesignTime>True</DesignTime>
```

Comment:
Question: should we create a different PR (or alternatively a commit) in order to track the re-formatting operations separately?

Reply:
These changes are unintended. They should be reverted.